dianlian-ai-runtime/src/dianlian_runtime/upload_inspection/service.py

```
@@ -0,0 +1,397 @@
+                for chunk in response.iter_raw():
+                    length += len(chunk)
+                    if length > MAX_CONTENT_LENGTH:
+                        raise UploadInspectionSourceConflict(
+                            "source object exceeds upload-policy-v1"
+                        )
+                    digest.update(chunk)
+                    target.write(chunk)
+        except (UploadInspectionSourceConflict, UploadInspectionUnsupportedMedia):
+            raise
+        except (httpx.HTTPError, OSError):
+            # httpx 异常可能包含签名 URL，禁止将原异常链带入日志或响应。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)